clippy_lints/src/literal_representation.rs

```diff
@@ -550,7 +569,7 @@ impl LiteralRepresentation {
             let f = &digits[0..1]; // first digit
             let m = &digits[1..digits.len() - 1]; // middle digits, except last
             let s = &digits[1..]; // suffix
-            // Powers of 2 with a margin of +15/-16
+                                  // Powers of 2 with a margin of +15/-16
```

Comment:
This indentation also seems wrong.

Reply:
I added a space to fix the spacing here too.